erddap/content/datasets.xml

```diff
@@ -13165,7 +13165,7 @@ AZOMP's sampling scheme off Atlantic Canada and the Labrador Sea involves annual
       </addAttributes>
     </dataVariable>
     <dataVariable>
-      <sourceName>=56.1833</sourceName>
+<sourceName>=-56.1833</sourceName>
```

Comment:
I'm not sure why GitHub is saying that this longitude will be turned negative since it is already negative within the main branch

https://github.com/cioos-atlantic/erddap-gold-standard/blob/63afd3976325cae40b305d99d33bf00497d8a3dc/erddap/content/datasets.xml#L13168

Reply:
Let's keep an eye to see what gets updated once merged